SkillChat.Client/Views/SendMessageControl.xaml.cs

```diff
@@ -7,15 +7,37 @@
 namespace SkillChat.Client.Views
 {
 	public class SendMessageControl : UserControl
-	{
+    {
+        private TextBox tb_message;
+        private bool isCursorSet;
         public SendMessageControl()
 		{
 			this.InitializeComponent();
+			tb_message = this.Get<TextBox>("InputMessageTB");
+            tb_message.LayoutUpdated += Tb_message_LayoutUpdated;
 		}
 
-		private void InitializeComponent()
+        private void InitializeComponent()
 		{
 			AvaloniaXamlLoader.Load(this);
 		}
+
+        private void Tb_message_LayoutUpdated(object sender, EventArgs e)
```

Comment:
Название нарушает общепринятую конвенцию именования
```suggestion
        private void MessageTextBoxLayoutUpdated(object sender, EventArgs e)
```